### DISCUSSION TITLE
3021 years to win??? 🤣🤣

### DISCUSSION THREAD

**Discussion by kitt3000black** [Chia-Network/chia-blockchain#6074] [General]
hi there

Soo. . . I think this farming think is about drive speed ( the general idea ), but at my 14H to now 19H the dificulty jump from 182 to 772 😮😮
![191990349_3443650032401353_3470997132918023540_n](https://user-images.githubusercontent.com/70859568/119711581-9b305400-be57-11eb-9d51-8d1b9834ff1a.jpg)
dificulty 182

![q](https://user-images.githubusercontent.com/70859568/119711719-bdc26d00-be57-11eb-86ec-b1a27e40e269.png)
dificulty 772

![無題](https://user-images.githubusercontent.com/70859568/119712139-345f6a80-be58-11eb-9d8d-134ca59c4434.png)
and least 3021 years to win???

**Comment by mkdev-code** [comment]
even with 100 plots there is no chance very low     

and hpool users farm with xch1f0ryxk6qn096hefcwrdwpuph2hm24w69jnzezhkfswk0z2jar7aq5zzpfj  walled  40% chance there 

Summary
Won 34.43% of the last 10,000 blocks  MONOPOL
